cms/check_rule_counts

```diff
@@ -148,44 +152,117 @@ if __name__ == '__main__':
 
-            stuck_locks_older_than_X_labels = ['Stuck_cnt']
-            replicating_locks_older_than_X_labels = ['Replicating_cnt']
+            # Number of Suspended rules older than x
 
-            stuck_locks_older_than_X_labels.extend(extra_prom_labels.keys())
-            replicating_locks_older_than_X_labels.extend(extra_prom_labels.keys())
+            query = (session.query(func.count(models.ReplicationRule.id))
+                     .filter(models.ReplicationRule.state == RuleState.SUSPENDED)
+                     .filter(models.ReplicationRule.created_at <= timeLimit))
+            result = query.scalar() or 0
+            print('rules.no_of_rules.suspended.%s.suspended_rules_cnt %s' % (a_name, result))
 
-            stuck_locks_older_than_gauge = Gauge('rucio_stuck_cnt_%s' % a_name, 'Number of stuck files older than X',
-                                                 labelnames=stuck_locks_older_than_X_labels,
-                                                 registry=registry)
-            replicating_locks_older_than_gauge = Gauge('rucio_replicating_cnt_%s' % a_name,
-                                                       'Number of replicating files older than X',
-                                                       labelnames=replicating_locks_older_than_X_labels,
-                                                       registry=registry)
+            prom_labels = {'age': a_name.split('_')[1]}
+            prom_labels.update(extra_prom_labels)
+            suspended_rules_older_than_gauge.labels(**prom_labels).set(result)
+            print(prom_labels, result)
+
+            monitor.record_gauge(name='rules.no_of_rules.suspended.%s.sum_locks_stuck_cnt' % a_name, value=result)
+
+            # Number of Stuck rules older than x
+
+            query = (session.query(func.count(models.ReplicationRule.id))
+                     .filter(models.ReplicationRule.state == RuleState.STUCK)
+                     .filter(models.ReplicationRule.created_at <= timeLimit))
+            result = query.scalar() or 0
+            print('rules.no_of_rules.stuck.%s.stuck_rules_cnt %s' % (a_name, result))
+
+            prom_labels = {'age': a_name.split('_')[1]}
+            prom_labels.update(extra_prom_labels)
+            stuck_rules_older_than_gauge.labels(**prom_labels).set(result)
+            print(prom_labels, result)
+
+            monitor.record_gauge(name='rules.no_of_rules.stuck.%s.sum_locks_stuck_cnt' % a_name, value=result)
+
+            # Number of replicating rules older than x
+
+            query = (session.query(func.count(models.ReplicationRule.id))
+                     .filter(models.ReplicationRule.state == RuleState.REPLICATING)
+                     .filter(models.ReplicationRule.created_at <= timeLimit))
+            result = query.scalar() or 0
+            print('rules.no_of_rules.replicating.%s.replicating_rules_cnt %s' % (a_name, result))
+
+            prom_labels = {'age': a_name.split('_')[1]}
```

Comment:
This seems to be a repeat (with no change in intermediate value of the code on 217 and 202 (a_name and extra_prom_labels are not changing)